client/lib/js/react/components/bigchaindb_ledgerplugin.js

```diff
@@ -122,12 +122,28 @@ class BigchainDBLedgerPlugin extends EventEmitter2 {
     getConnectors() {
     }
 
+    /*
+    Initiates a ledger-local transfer.
+     */
     send(transfer) {
         return co.wrap(this._send).call(this, transfer);
     }
 
     * _send(transfer) {
-        
+        let res;
+        try {
+            res = yield request('assets_escrow', {
```

Comment:
Ah, @TimDaub had a comment in the last PR that was about making the request signature non-throwing. I can make a change to the `request` module so that this can just be `return yield request(...).catch(() => throw new Error(...));`